04-sop_guidelines.Rmd

```diff
@@ -9,26 +9,22 @@ ottrpal::set_knitr_image_path()
 
 This chapter will cover:  
 
-- Introduction to the CIViC data model
-- Demo of the CIViC web interface
+- AMP/ASCO/CAP Guidelines
+- ClinGen/CGC/VICC Oncogenicity
 
-## Introduction to the CIViC data model
+## AMP/ASCO/CAP Guidelines
 
-For background on the CIViC resource and data model please consult the CIViC core papers [@Griffith2017; @Krysiak2022; @Krysiak2022b].
+Guidelines for the interpretation of clinically actionable somatic variants have been established by AMP/ASCO/CAP [@Li2017], ESMO [@Mateo2018] and others. The AMP/ASCO/CAP guidelines were published in 2017, with the aim of standardizing the interpretation and reporting of molecular results among clinical laboratories. These guidelines focus on the predictive/Therapeutic, Diagnostic, and Prognostic evidence categories discussed above. The guidelines introduce a system of tiering somatic variants based on clinical knowledge associated to the variant. The tiering system considers the state of knowledge and clinical actionablilty of a somatic variant within a specific cancer type, along with an associated drug if applicable. Briefly, Tier I Level A variants are established in guidelines such as those provided by the FDA or WHO. For example, in the presence of the EGFR L858R variant, the tyrosine kinase inhibitor erlotinib is approved for use in non small cell lung cancer, and therefore in this context the variant is Tier I Level A. Tier I Level B variants are those where a reasonable amount of knowledge is present for the variant and disease in the form of clinical trials, but approvals are not in place. If regulatory approvals become established, then the variant will be upgraded to Tier I Level A. Tier II Level C variants are those where the state of knowledge has not proceeded beyond case studies. This Tier also applies to cases where a drug which is established for targeted use against a given variant in a certain cancer type is applied against the same variant but in a different cancer type. Tier II Level B variants are those where studies have not yet progressed beyond preclinical work. Note that besides targetable driver variants, these tiers can also be applied to resistance variants, as this variant type also appears in guidelines.
```

Comment:
```suggestion
Guidelines for the interpretation of clinically actionable somatic variants have been established by AMP/ASCO/CAP [@Li2017], ESMO [@Mateo2018] and others. The AMP/ASCO/CAP guidelines were published in 2017 with the aim of standardizing the interpretation and reporting of molecular results among clinical laboratories. These guidelines focus on the Predictive/Therapeutic, Diagnostic, and Prognostic evidence categories discussed above. The guidelines introduce a system of tiering somatic variants based on clinical knowledge associated to the variant. The tiering system considers the state of knowledge and clinical actionablilty of a somatic variant within a specific cancer type, along with an associated drug if applicable. Briefly, Tier I Level A variants are established in guidelines such as those provided by the FDA or WHO. For example, in the presence of the EGFR L858R variant, the tyrosine kinase inhibitor erlotinib is approved for use in non small cell lung cancer, and therefore in this context the variant is Tier I Level A. Tier I Level B variants are those where a reasonable amount of knowledge is present for the variant and disease in the form of clinical trials, but approvals are not in place. If regulatory approvals become established, then the variant will be upgraded to Tier I Level A. Tier II Level C variants are those where the state of knowledge has not proceeded beyond case studies. This Tier also applies to cases where a drug which is established for targeted use against a given variant in a certain cancer type is applied against the same variant but in a different cancer type. Tier II Level B variants are those where studies have not yet progressed beyond preclinical work. Note that besides targetable driver variants, these tiers can also be applied to resistance variants, as this variant type also appears in guidelines.
```
It might also be helpful to replace "above" with something like "the previous chapter" or to refer to the section by name or link.